src/main/scala/com/databricks/spark/avro/DefaultSource.scala

```diff
@@ -210,6 +221,8 @@ private[avro] class DefaultSource extends FileFormat with DataSourceRegister {
 private[avro] object DefaultSource {
   val IgnoreFilesWithoutExtensionProperty = "avro.mapred.ignore.inputs.without.extension"
 
+  val AVRO_SCHEMA = "avroSchema"
```

Comment:
@clockfly can you address this comment? Let's make the variables of `IgnoreFilesWithoutExtensionProperty` and `AVRO_SCHEMA` consistent. Thanks!